android/app/src/main/kotlin/org/getlantern/lantern/util/PaymentsUtil.kt

```diff
@@ -25,6 +27,14 @@ import org.getlantern.mobilesdk.Logger
 
 class PaymentsUtil(private val activity: Activity) {
     private val session: LanternSessionManager = LanternApp.getSession()
+    private var paymentDialog: ProgressDialog = ProgressDialog(activity)
```

Comment:
Not super important, but I think we removed use of ProgressDialog because.. 

> "ProgressDialog is a modal dialog, which prevents the user from interacting with the app. Instead of using this class, you should use a progress indicator like [ProgressBar](https://developer.android.com/reference/android/widget/ProgressBar), which can be embedded in your app's UI." 

https://developer.android.com/reference/android/app/ProgressDialog

Is it possible to show the loading spinner on the Flutter side the way we do with some other Pro calls, or is there a reason to prefer this method instead?

Reply:
Thanks for letting me know, Done, Moved to the flutter side.